3-extensions/serialization/dubbo-samples-serialization/dubbo-samples-serialization-java/pom.xml

```diff
@@ -66,103 +83,34 @@
                 <artifactId>dubbo-dependencies-zookeeper-curator5</artifactId>
```

Comment:
I think this dependency can be deleted now.

Reply:
Thanks for the suggestion, this dependency has been removed!